extensions/resource-deployment/package.json

```diff
@@ -13,7 +13,7 @@
     "azdata": "^1.6.0"
   },
   "activationEvents": [
-    "onCommand:azdata.resource.create"
```

Comment:
Is there a reason we are changing this activation event? I don't see anything about this PR that requires it.